libxmp/exempi.py

```diff
@@ -69,7 +69,32 @@ def _load_exempi():
 
     return EXEMPI
 
-EXEMPI = _load_exempi()
+
+class LazyExempi:
+    """This class is the wrapper for the Exempi lib which
+    prevents the crash while building/importing the exempi.py
+    module if the Exempi is not installed on the machine, essentially
+    making Exempi a run-time dependency.
+    """
+    def __init__(self):
+        self._exempi = None
+
+    def __getattr__(self, attr):
+        if self._exempi is None:
+            self.load()
```

Comment:
If load is only used once, we can inline it

Reply:
OK